src/avx2/stage1.rs

```diff
@@ -107,9 +137,10 @@ fn find_odd_backslash_sequences(input: &SimdInput, prev_iter_ends_odd_backslash:
     // should be flipped
     let (mut odd_carries, iter_ends_odd_backslash) = bs_bits.overflowing_add(odd_starts);
 
-    odd_carries |= *prev_iter_ends_odd_backslash; // push in bit zero as a potential end
-                                                  // if we had an odd-numbered run at the
-                                                  // end of the previous iteration
+    odd_carries |= *prev_iter_ends_odd_backslash;
+    // push in bit zero as a potential end
```

Comment:
This comment was attached to the line above (`odd_carries`) I think if we remove the attachment we should move the comment before the code not after, otherwise the next person reading it might get confused where it belongs to.

Reply:
ugh rustfmt

Reply:
Ja it's a PITa on the bright side it helps keeping things coherent and no one is fighting about how things are formatted  ("please add an empty newline at the end of each file" :joy:)